### PR TITLE
Make class versatile with clsx

### PR DESCRIPTION
**Pull request by PuruVJ**
- Add clsx to make `class` prop versatile
- Fix the issue where style object containing camel case properties doesn't work

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -32,8 +33,14 @@ export const style = (node, styleData = {}) => {
...
-            for (const [property, value] of Object.entries(styleData))
-                pseudoElement.style.setProperty(property, value);
+            for (const [property, value] of Object.entries(styleData)) {
+                // Do a setProperty in case it's a CSS variable
+                if (property.startsWith('--')) {
+                    pseudoElement.style.setProperty(property, value);
+                } else {
+                    pseudoElement.style[property] = value;
+                }
+            }
```
Comment (ghostdevv) [inline src/actions.js:43]: Can you mention this change in pr

File: dev/src/App.svelte [hunk 2]
```diff
@@ -1,10 +1,12 @@
 <script lang="ts">
     // import logo from './assets/svelte.png';
     import Counter from './lib/Counter.svelte';
-    import { style } from 'svelte-body';
+    import { style, classList } from '../../dist';
```
Comment (ghostdevv) [inline dev/src/App.svelte:4]: But the workspace----

Comment (PuruVJ) [reply dev/src/App.svelte:4]: I dunno how to set it up 🥲🥲

Comment (ghostdevv) [reply dev/src/App.svelte:4]: I set it up @PuruVJ

**Pull request merged**